src/System.Private.CoreLib/shared/System/Guid.cs

```diff
@@ -427,7 +427,7 @@ private static bool TryParseGuid(ReadOnlySpan<char> guidString, GuidStyles flags
             }
 
             // Check for dashes
-            bool dashesExistInString = guidString.IndexOf('-') >= 0;
+            bool dashesExistInString = guidString.Contains('-');
```

Comment:
There are other places we could use the Contains method. For example:
https://github.com/dotnet/coreclr/blob/91820bdd42711266faae1c15d2176b9f47e48efe/src/System.Private.CoreLib/shared/System/Guid.cs#L452
https://github.com/dotnet/coreclr/blob/91820bdd42711266faae1c15d2176b9f47e48efe/src/System.Private.CoreLib/shared/System/Guid.cs#L474
https://github.com/dotnet/coreclr/blob/91820bdd42711266faae1c15d2176b9f47e48efe/src/System.Private.CoreLib/shared/System/IO/PathHelper.Windows.cs#L36
https://github.com/dotnet/coreclr/blob/91820bdd42711266faae1c15d2176b9f47e48efe/src/System.Private.CoreLib/shared/System/String.Searching.cs#L27

Reply:
Will add these to the new PR.